doc/installing.rst

```diff
@@ -93,48 +117,28 @@ Some users have run into the following complications when installing the
    llvm version 9.0.1 using `Homebrew <https://brew.sh/>`_ and then add
    environmental variables to your shell rc that point to the reverted
    directory structure.
- 
- - At the moment, an implicit dependency on PyQt5 remains (in addition to
-   PySide2) because of our dependence on ``linetools``.
 
  - Note that ``shapely`` is listed as an optional dependency, but is only
    currently used by one method that calculates the spaxel area for KCWI
    output datacubes.
 
+ - Note that you will likely have either PyQt5 or PySide2 installed and PypeIt requires
+   one of these two packages installed for any of the GUIs to work, in particular
+   ``pypeit_show_2dspec`` and ``pypeit_show_1dspec``. If you have neither installed, and
+   both ``pypeit_show_2dspec`` and ``pypeit_show_1dspec`` crash, then you should install
+   one of PySide2 or PyQt5 (``pip install pyside2`` or ``pip install pyqt5`` or
+   ``conda install pyside2`` or ``conda install pyqt``).
+
  - For the developer-only (``Sphinx``) packages, download
    `requirements_doc.txt <https://github.com/pypeit/PypeIt/blob/master/requirements_doc.txt>`_
-   and install with ``pip install -r requirements_doc.txt``.
+   and install with ``conda install -y -c conda-forge --file requirements_doc.txt`` or ``pip install -r requirements_doc.txt``.
 
 ----
 
-Installing PypeIt
-=================
-
-Please read all of the text in this sub-section before choosing which
-of the two methods described below for how you wish to install
-``PypeIt``. Once you've completed the installation steps, you should
-:ref:`test_installation`.
-
-Install using pip
------------------
-
-If you are not using code on the edge of development, then
-we recommend that you install ``PypeIt`` with ``pip``::
-
-    pip install pypeit
-
-This has been known to fail on some systems (and we're working to fix
-the issue). If you have problems, instead try::
-
-    pip install git+https://github.com/pypeit/PypeIt.git
-
-If that also fails, please `submit an issue
-<https://github.com/pypeit/PypeIt/issues>`__.
-
-Install from the git source
----------------------------
+Install from the git source (Advanced)
+======================================
 
-If ``pip`` is unsuccessful or if you are planning to use any of the
+If ``conda`` and ``pip`` are unsuccessful, or if you are planning to use any of the
 ``PypeIt`` development branches, then you should install directly
 from GitHub.
 
```

Comment:
@finagle29 this is different from the procedure I had to follow to get this to work, specifically the yaml file containing the dependencies which get installed is not mentioned here. Specifically I had to run: 

conda env create -f environment.yml 
conda activate pypeit 
pip install pyqt5
conda activate pypeit 
pip install -e .

Can you please add a note on this. You can state that if the users prefer to install the dependencies themselves via conda or pip they can also do that, but the yaml file should be mentioned here. 

Thanks!

Reply:
I think I addressed this in my latest commit, could you take a look at it?